linera-execution/src/execution_state_actor.rs

```diff
@@ -508,8 +522,8 @@ impl Debug for Request {
                 .field("url", url)
                 .finish_non_exhaustive(),
 
-            Request::FetchJson { url, .. } => formatter
-                .debug_struct("Request::FetchJson")
+            Request::HttpPost { url, .. } => formatter
+                .debug_struct("Request::HttpPost")
                 .field("url", url)
```

Comment:
We could add the content type to the debug output.

Reply:
Ok, done.